gcode.c

```diff
@@ -971,23 +971,20 @@ uint8_t gc_execute_line(char *line)
           mc_arc(gc_state.position, gc_block.values.xyz, gc_block.values.ijk, gc_block.values.r, 
             gc_state.feed_rate, gc_state.modal.feed_rate, axis_0, axis_1, axis_linear, gc_block.values.n);  
           break;
-        case MOTION_MODE_PROBE: {
-          uint8_t sensor;
-          if (bit_istrue(value_words, bit(WORD_P))) {
-            // Cast the p parameter to uint8_t for the sensor address
-            sensor = (uint8_t)gc_block.values.p;
-            bit_false(value_words, bit(WORD_P));
-          } else {
-            // If gc_block.values.p is not defined, select the carousel
-            // magazine alignment sensor as the default. This allows for
-            // backward compatability with motion.
-            sensor = MAG_SENSOR; 
-          }
+        case MOTION_MODE_PROBE:          
           probe_move_to_sensor(gc_block.values.xyz, gc_state.feed_rate,
-          gc_state.modal.feed_rate, gc_block.values.n, sensor); 
+          gc_state.modal.feed_rate, gc_block.values.n, gc_block.values.p); 
           retval = STATUS_QUIET_OK;
           //block.values.xyz is updated inside probe_cycle, so the next line is correct
-        }
+      
```

Comment:
Regarding the specifics of the comment - I didn't write that comment. 

It looks like the comment is related to the commented lines of code following it in which it checks if we are already at the position to which we try to probe to and if the probe is already triggered.